src/style.css

```diff
@@ -1,3 +1,74 @@
+@font-face {
+  font-family: Roboto;
+  src: url(fonts/Roboto/Roboto-Medium.ttf);
+}
+
 body {
   margin: 0;
 }
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  display: block;
+  float: right;
+}
+
+li a {
+  text-align: center;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav {
+  flex-direction: row-reverse;
+}
+
+.link-8,
+.link-5 {
+  color: #00acdc;
+}
+
+.link {
+  text-decoration: none;
+  color: black;
+  font-family: Roboto, sans-serif;
+  height: 60px;
+  display: flex;
+  font-size: 12px;
+  text-align: center;
+  align-items: center;
+  justify-content: center;
+  padding: 0 20px;
+}
+
+.link:hover {
+  color: #00acdc;
+}
+
+.link-8::after {
+  content: '';
+  display: block;
+  position: relative;
```

Comment:
try to use absolute and relative for link ([you can read more about positioning here](https://www.freecodecamp.org/news/css-positioning-position-absolute-and-relative/))